### PR TITLE
EL-1018: Update CW Form template used in pre-population

### PR DESCRIPTION
**Pull request by MazOneTwoOne**
[Jira ticket](https://dsdmoj.atlassian.net/browse/EL-1018)

## What changed and why

Updating CW Form PDFs that are populated by CCQ.

## Guidance to review

Removed some tests, which talk about, when the 'MTR Phase 1' feature flag is enabled.  Wasn't sure if I should also remove the mappings & PDF templates for 'MTR Phase 1' (and associated tests)??

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing
- Branch is generally up to date with main Github - definitely no conflicts
- No unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- PR description says *what* changed and *why*, with a link to the JIRA story.
- Diff has been checked for unexpected changes being included.
- Commit messages say why the change was made.

### CODE REVIEW

**Review by patrick-laa — COMMENTED** [review]
We should remove the things we're no longer using - that means removing PDFs that are no longer ever server, removing mappings that are no longer ever read, and removing redundant entries in the ` TEMPLATES ` hash.

File: app/lib/controlled_work_mappings/cw2_header.yml [hunk 1]
```diff
@@ -0,0 +1,261 @@
+# This file defines the mappings used to prepopulate a CW2 IMM PDF form with the answers given by a user to CCQ's questions
```
Comment (patrick-laa) [inline app/lib/controlled_work_mappings/cw2_header.yml:1]: This alarms me - GitHub hasn't picked up that this file is just `cw2_mtr_phase_2.yml` renamed, although it has done for every other new yaml file. Are you confident that this is identical to the file you've deleted?

File: app/services/controlled_work_document_value_mapping_service.rb [hunk 2]
```diff
@@ -25,7 +25,7 @@ def call(session_data, mappings)
     def convert(mapping, content)
       case mapping[:type]
       when "checkbox"
```
Comment (patrick-laa) [inline app/services/controlled_work_document_value_mapping_service.rb:27]: so yeah, as per above, I'd change this to `always_checked_checkbox` or something now that its behaviour has changed.

**Review by patrick-laa — COMMENTED** [review]
I think we're basically there, just a bit nervous with all these yaml files flying around.

File: app/lib/controlled_work_mappings/cw2_header.yml (hunk 1 above)
Comment (MazOneTwoOne) [reply app/lib/controlled_work_mappings/cw2_header.yml:1]: I can explain - it was my 'neatness' itch, that I had to scratch. 

This **was** copied from the previous `cw2_mtr_phase_1.yml` file. I just indented mappings to the left, to match all the other mappings.

**Pull request merged**